src/SFA.DAS.EmployerFinance.Api/Program.cs

```diff
@@ -15,6 +16,10 @@ public static void Main(string[] args)
             WebHost.CreateDefaultBuilder(args)
                 .UseKestrel(options => options.AddServerHeader = false)
                 .UseStartup<Startup>()
-                .UseStructureMap();
+                .UseStructureMap()
+                .ConfigureServices((context, collection) =>
+                {
+                    collection.AddApplicationInsightsTelemetry(context.Configuration);
```

Comment:
We're not passing in any custom configuration so we can call `UseApplicationInsights` on the `IWebHostBuilder` instead.